integration/terraform/ec2/localstack/README.md

```diff
@@ -0,0 +1,19 @@
+The localstack ec2 will bring up an instance of localstack on an ec2 instance to be used by other tests
+
+Use the ubuntu ami
+
+Instance assumptions
+1. Use the same ami as the ubuntu integration test (if you use this ami you can ignore the other assumptions)
+2. docker
+3. docker-compose
+4. git
+5. openssl
+6. aws-cli
+7. CloudWatchAgentServerRole is attached
+8. crontab
+
+Tag the ec2 instance with LocalStackIntegrationTestInstance
+
+There is no guarantee of destroy if the pipeline state is cancelled. Since create happens before tests job and cancel happens after test job. 
```

Comment:
Can you clarify on this? Are you saying that if the pipeline is cancelled, we will have orphaned EC2 instances in the account that won't be cleaned up? If so, I think we should add to the cleaner (not part of this PR) to also clean up hosts used by these integration tests that are stale.

Reply:
I re read the gh docs and it seems this my understanding was incorrect. Will delete this.